dev/com.ibm.ws.microprofile.metrics.2.3/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -141,6 +141,11 @@ grpc.client.receivedMessages.total.description=The total number of stream messag
 grpc.client.responseTime.total.description=The total response time of completed RPCs.
 
 REST.request.description=The number of invocations and total response time of this RESTful resource method since the start of the server.
+
+requestTiming.requestCount.description=The number of servlet requests since the server started.
+requestTiming.activeRequestCount.description=The number of servlet requests currently running.
+requestTiming.slowRequestCount.description=The number of servlet requests that are currently running and are deemed slow.
+requestTiming.hungRequestCount.description=The number of servlet requests that are currently running and are deemed hung.
```

Comment:
Suggested change: Replace "and are deemed hung" with "but are hung" or "but are unresponsive"

Reply:
fixed